.github/workflows/bump-version.yaml

```diff
@@ -0,0 +1,14 @@
+name: "🆙 Bump Version"
+
+on:
+  release:
+    types:
+      - published
+  workflow_dispatch:
+
+jobs:
+  call:
+    uses: jellyfin/jellyfin-meta-plugins/.github/workflows/bump-version.yaml@master
+    with:
+      csproj-name: Jellyfin.Plugin.Vgmdb
```

Comment:
Assuming your structure cleanup will move and rename the csproj to this:
`Jellyfin.Plugin.Vgmdb/Jellyfin.Plugin.Vgmdb.csproj`

This will work then :+1: